agent_build/tools/files_checksum_tracker.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2014-2021 Scalyr Inc.
```

Comment:
@Kami This new file mostly contains logic that has been moved from the `deployments.py` module.